File: Chain of Responsibility Pattern/Consultant.java
package consultant;

import patient.Patient;
import diseaseseverity.DiseaseSeverity;
import handler.Handler;

public class Consultant extends Handler{
	
	@Override
	public void handlePatient(Patient patient){
		
		if(patient.getDiseaseSeverity() == DiseaseSeverity.CRITICAL){
			System.out.println("Diagnose the Patient " + patient.getName() + " by the Consultant");
		}
	
	}
	
}

File: Design Patterns/Chain of Responsibility Pattern in JavaScript/Consultant.js
const Handler = require('./Handler');
const DiseaseSeverity = require('./DiseaseSeverity');

class Consultant extends Handler {

    handlePatient(patient) {

        super.handlePatient(patient);

        if (patient.diseaseSeverity === DiseaseSeverity.CRITICAL) {
            console.log("Diagnose the Patient " + patient.name + " by the Consultant \n");
        }

    }

}

module.exports = Consultant;